contracts/icns-registrar/src/contract.rs

```diff
@@ -264,10 +267,17 @@ pub fn execute_claim(
     }
 
     let verifying_msg: VerifyingMsg = from_slice(verifying_msg_str.as_bytes())?;
+    // NL: This is probably not a problem, but just noting that this does not
+    // check for existance and instead overrides the existing id with the new
+    // name. Meaning that you can claim more than once. Should be ok since the
+    // name is checked in the message validation
```

Comment:
Oh this could be problem, we are making sure that 1 twitter id can claim once to avoid "claim -> changes username -> claim again" case